app/views/curation_concerns/base/_form_required_information.html.erb

```diff
@@ -1,3 +1,7 @@
+<% unless params[:parent_id].blank? %>
+  <%= f.hidden_field :parent_id, value: params[:parent_id] %>
```

Comment:
Is this necessary? I would assume the form would POST to the nested create action, which would have parent_id